documentation/modules/operators/con-topic-replication.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='ref-topic-replication-{context}']
+= Configuring topics for replication and number of partitions
+
+The recommended configuration for topics managed by the Topic Operator is a topic replication factor of 3, and a minimum of 2 in-sync replicas.
+
+[source,shell,subs="+attributes"]
+----
+apiVersion: {KafkaTopicApiVersion}
+kind: KafkaTopic
+metadata:
+  name: my-topic
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  partitions: 10 # <1>
+  replicas: 3 # <2>
+  config:
+    min.insync.replicas: 2 # <3>
+  #...
+----
+<1> The number of partitions for the topic.
+<2> The number of replica topic partitions. Currently, this cannot be changed in the `KafkaTopic` resource, but it can be changed using the `kafka-reassign-partitions.sh` tool.
+<3> The minimum number of replica partitions that a message must be successfully written to, or an exception is raised.
+
+NOTE: In-sync replicas are used in conjunction with the `acks` configuration for producer applications.
+The `acks` configuration determines the number of follower partitions a message must be replicated to before the message is acknowledged as successfully received.
+The Topic Operator runs with `acks=all`, whereby messages must be acknowledged by all in-sync replicas.
```

Comment:
I think this is misleading. While it's true for how the topic store mechanism works for the BTO it could be easily misinterpreted by a novice user to mean "when you use the TO your applications will be using `acks=all`, which is not the case.

```suggestion
The bidirectional Topic Operator runs with `acks=all` for its internal topics whereby messages must be acknowledged by all in-sync replicas.
```